WordPress/openverse-frontend
# Make the homepage search input border transparent

## Fixes


Fixes #1579 by @panchovm

## Description


This PR uses Tailwind's `!` important modifier to make sure that the transparent color of the homepage search input is not overridden by `VInputField`.

### Before:
<img width="628" alt="Search input that has a border" src="https://user-images.githubusercontent.com/15233243/184321062-c35d4913-c0cd-4a3c-a944-49750aa1a0fb.png">

### After:
<img width="595" alt="Search input with a transparent border" src="https://user-images.githubusercontent.com/15233243/184321114-e728366f-3fb4-4e58-9327-bad5fad652ab.png">


## Testing Instructions

Run the app and look at the homepage. There should be no `charcoal-20` border around the search bar in the resting state.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
